### ISSUE TITLE
Load times for accessions with a large number of files

### ISSUE THREAD

**Issue by awead** [psu-stewardship/archivesphere#160]
Accessions with a large amount of files are loading very slowly. At present, this [accession](https://archivesphere.dlt.psu.edu/accessions/cc08hf69f) takes ~5 minutes to load and contains 33,659 files. The load time is the same from 1.1.3 release as well as 2.0.0, so this is not a new problem. However, the 2.0.0 release load time was calculated without 3656ad82d329382bc7e5d93ce3149968500db683 in effect. That commit would make the time even longer.

One unexpected effect that is not present in the 1.1.3 release is that browsing accessions in the catalog view, page by page, is also much slower if the result includes the above accession in the set of results. This seems doubly odd because  file membership shouldn't be accounted for when viewing accessions at the index level, and should only occur at the individual show level.

**Comment by hectorcorrea** [comment]
The user of Solr Joins might help but that does not seem to be the bottleneck on this ticket. I hardcoded the `bytes` method (indicated here https://github.com/narogers/sufia/commit/b2625e956efd98c9183d0c7d2407615df12a6c96) to return always 32 and loading the large collection indicated in this ticket still takes 5 minutes.

**Comment by awead** [comment]
Interesting. I wonder if it's just ActiveFedora loading up all the objects in the collection. Have you looked at the Fedora log to see if that's the case? @cam156 might know this, but I thought there was some change made to hydra-collections that fixed this, but I can't remember which version that was in.

**Comment by hectorcorrea** [comment]
So far this is what I've found:
* 2 minutes spend **before** it even gets to the show method of the controller. Not sure where this time is going. I removed the `before_filter` from the controller and made no difference. 
* 1 minute producing the tree
* 2 minutes rendering the views

I might be able to shave sometime from the code that produces the tree and renders the view. I suspect this code is the one that it's making calls to Fedora but I haven't gotten that far.
